src/styles/globals.css

```diff
@@ -78,17 +97,64 @@ textarea {
   outline: none;
 }
 
-/* Breakpoints recommended by Bootstrap */
-/* Custom, iPhone Retina */
-@media only screen and (min-width: 320px) {
+/* Scalable H elements with h classes that can be a substitute for H elements */
+h1,
+.h1 {
+  font-size: var(--h1);
+}
+
+h2,
+.h2 {
+  font-size: var(--h2);
+}
+
+h3,
+.h3 {
+  font-size: var(--h3);
+}
+
+h4,
+.h4 {
+  font-size: var(--h4);
+}
+
+h5,
+.h5 {
+  font-size: var(--h5);
 }
 
-/* Extra Small Devices, Phones */
+h6,
+.h6 {
+  font-size: var(--h6);
+}
+
+.sm {
+  font-size: var(--sm);
+}
+
+.xs {
+  font-size: var(--xs);
+}
+
+p {
+  font-size: var(--base-size);
+}
```

Comment:
this is ok as a stopgap since we've already used `<h*/>` elements everywhere but might be a bit better to define these sizes in MUI's theme (I think they might already be there) and use `<Typography/>` elements

Reply:
I'm not familiar with the MUI theme and how defining it can override these elements. In any case, these can be used as fallback if the MUI theme is not defined right?

Reply:
yep, just trying to avoid duplicated definitions